.travis.yml

```diff
@@ -1,11 +1,22 @@
 ---
 language: node_js
+node_js:
+  - "4"
+  - "6"
+  - "stable"
 
 sudo: false
 
 cache:
-  directories:
-    - node_modules
+  yarn: true
+
+before_script:
+  # ensure we are testing against latest versions of all deps when using Yarn
+  - yarn upgrade
+
+branches:
+  only:
+    - master
```

Comment:
I'd rather just not worry about it.  I updated this to match what ember-cli itself does: https://github.com/ember-cli/ember-cli/blob/master/.travis.yml#L9-L15.